client/src/components/QuestionsAndAnswers/QAList/QAListItem/index.jsx

```diff
@@ -3,17 +3,51 @@ import AnswerList from './AnswerList';
 import AddAnswer from './AddAnswer';
 import Helpful from './Helpful';
 
-const QAListItem = () => {
+const QAListItem = ({question}) => {
+  const [answerList, setAnswerList] = React.useState([]);
+  const [displayedAnswerList, setDisplayedAnswerList] = React.useState([]);
+  const [displayCount, setDisplayCount] = React.useState(0);
+  const [helpfulCount, setHelpfulCount] = React.useState(question?.helpfulCount);
+
+  React.useEffect(() => {
+    console.log('First A Render')
+    //set the answerList and displayedAnswerList on page load after an API call
+  });
```

Comment:
Need an empty dependency array here if you intend to make an API call here at initial render.